Author: Reinhard Tartler <siretart@tauware.de>
Description: Allow compilation with earlier go-crio
Forwarded: no

Updating crio was requested in #993317. Please drop this patch after version
5.1.0 has reached unstable

diff --git a/pkg/criu/criu.go b/pkg/criu/criu.go
index 2a6805979..d6eec7023 100644
--- a/pkg/criu/criu.go
+++ b/pkg/criu/criu.go
@@ -1,7 +1,7 @@
 package criu
 
 import (
-	"github.com/checkpoint-restore/go-criu/v5"
+	"github.com/checkpoint-restore/go-criu"
 )
 
 // MinCriuVersion for Podman at least CRIU 3.11 is required
